cli/mender-artifact/modify.go

```diff
@@ -82,7 +75,7 @@ func modifyArtifact(c *cli.Context) error {
 	if isArtifact {
 		// re-create the artifact
 		err := repackArtifact(comp, c.Args().First(), modifyCandidates[0].path,
-			c.String("key"), c.String("name"))
+			"", c.String("name"))
```

Comment:
There is only one other instance of this function call too, in `partition.go`. I *think* it is being used during `cp` stage. Have you checked the behavior of signing key there? Maybe it should be removed from both places, and the function argument `key` removed, like the previous one.

Reply:
Thanks, I will look into it in detail and see if we can remove it

Reply:
Fixed. See last commit